app/assets/javascripts/discourse/tests/acceptance/sidebar-anonymous-community-section-test.js

```diff
@@ -53,24 +128,6 @@ acceptance("Sidebar - Anonymous user - Community Section", function (needs) {
       I18n.t("sidebar.sections.community.links.everything.content"),
       "displays the everything section link first"
     );
-
-    assert.strictEqual(
-      sectionLinks[1].textContent.trim(),
-      I18n.t("sidebar.sections.community.links.users.content"),
-      "displays the users section link second"
-    );
-
-    assert.strictEqual(
-      sectionLinks[2].textContent.trim(),
-      I18n.t("sidebar.sections.community.links.about.content"),
-      "displays the about section link third"
-    );
-
-    assert.strictEqual(
-      sectionLinks[3].textContent.trim(),
-      I18n.t("sidebar.sections.community.links.faq.content"),
-      "displays the FAQ section link last"
-    );
```

Comment:
Just want to confirm why we're removing these assertions.

Reply:
Yes, before for anonymous users, about and faq links were in primary section and in more section for authenticated users.

We decided to unify it, and move to always to more. Those tests were added later in this part:

<img width="992" alt="Screenshot 2023-05-02 at 1 59 00 pm" src="https://user-images.githubusercontent.com/72780/235576535-94904cfc-a543-40c0-9767-553d9988abfb.png">

Reply:
Got it :+1: